baseapp/baseapp.go

```diff
@@ -674,7 +684,9 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte) (gInfo sdk.GasInfo, re
 	// and we're in DeliverTx. Note, runMsgs will never return a reference to a
 	// Result if any single message fails or does not have a registered Handler.
 	result, err = app.runMsgs(runMsgCtx, msgs, mode)
+
 	if err == nil {
+
```

Comment:
@kocubinski please. undo these changes in https://github.com/cosmos/cosmos-sdk/pull/13262 :)